AdjustTokenPrivileges failed log error when not running with elevated permissions

Hello,

After switching to 5.3.0/5.3.1 I get the error above, any idea ?

[ERROR] oshi.software.os.windows.WindowsOperatingSystem [main] AdjustTokenPrivileges failed. Error: 1300

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,7 @@
 5.3.2 (in progress)
 ================
 * [#1374](https://github.com/oshi/oshi/pull/1374): Revert logical processor indexing, fix counter instance naming - [@dbwiddis](https://github.com/dbwiddis).
+* [#1375](https://github.com/oshi/oshi/pull/1375): Reduce log level for expected Windows permission failures - [@dbwiddis](https://github.com/dbwiddis).
 * Your contribution here
 
 5.3.0 (2020-10-11), 5.3.1 (2020-10-18)

--- a/oshi-core/src/main/java/oshi/software/os/windows/WindowsOperatingSystem.java
+++ b/oshi-core/src/main/java/oshi/software/os/windows/WindowsOperatingSystem.java
@@ -422,8 +422,9 @@ public class WindowsOperatingSystem extends AbstractOperatingSystem {
     }
 
     /**
-     * Enables debug privileges for this process, required for OpenProcess() to get
-     * processes other than the current user
+     * Attempts to enable debug privileges for this process, required for
+     * OpenProcess() to get processes other than the current user. Requires elevated
+     * permissions.
      *
      * @return {@code true} if debug privileges were successfully enabled.
      */
@@ -445,10 +446,12 @@ public class WindowsOperatingSystem extends AbstractOperatingSystem {
             WinNT.TOKEN_PRIVILEGES tkp = new WinNT.TOKEN_PRIVILEGES(1);
             tkp.Privileges[0] = new WinNT.LUID_AND_ATTRIBUTES(luid, new DWORD(WinNT.SE_PRIVILEGE_ENABLED));
             success = Advapi32.INSTANCE.AdjustTokenPrivileges(hToken.getValue(), false, tkp, 0, null, null);
-            // Possible to partially succeed, only need to check LastError
             int err = Native.getLastError();
-            if (err != WinError.ERROR_SUCCESS) {
+            if (!success) {
                 LOG.error("AdjustTokenPrivileges failed. Error: {}", err);
+                return false;
+            } else if (err == WinError.ERROR_NOT_ALL_ASSIGNED) {
+                LOG.debug("Debug priveleges not enabled.");
                 return false;
             }
         } finally {

